packages/babel-parser/src/parse-error/to-node-description.js

```diff
@@ -0,0 +1,34 @@
+const NodeDescriptions = {
+  ArrayPattern: "array destructuring pattern",
+  AssignmentExpression: "assignment expression",
+  AssignmentPattern: "assignment expression",
+  ArrowFunctionExpression: "arrow function expression",
+  ConditionalExpression: "conditional expression",
+  ForOfStatement: "for-of statement",
+  ForInStatement: "for-in statement",
+  ForStatement: "for-loop",
+  FormalParameters: "function parameter list",
+  Identifier: "identifier",
+  ObjectPattern: "object destructuring pattern",
+  ParenthesizedExpression: "parenthesized expression",
+  RestElement: "rest element",
+  UpdateExpression: {
+    true: "prefix operation",
+    false: "postfix operation",
+  },
+  VariableDeclarator: "variable declaration",
+  YieldExpression: "yield expression",
+};
+
+type NodeTypesWithDescriptions = $Keys<typeof NodeDescriptions>;
```

Comment:
```suggestion
type NodeTypesWithDescriptions = $Keys<$Diff<typeof NodeDescriptions, { UpdateExpression: any }>>;
```

so that it does not include `UpdateExpression` and thus `{ type: "UpdateExpression" }` is not assignable to `NodeWithDescription` (because it must now have `prefix: boolean`).

Reply:
Done, I did it on a separate commit because I had to do the code formatting with Prettier: https://github.com/tolmasky/babel/commit/650917d5a1dca21369440d223e59730af02cff7f